datumaro/plugins/data_formats/segment_anything/importer.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import json
+import os
+from typing import Dict, List, Optional
+
+from datumaro.components.format_detection import FormatDetectionConfidence, FormatDetectionContext
+from datumaro.components.importer import Importer
+from datumaro.errors import DatasetImportError
+
+
+class SegmentAnythingImporter(Importer):
+    _N_JSON_TO_TEST = 10
+
+    @classmethod
+    def detect(
+        cls,
+        context: FormatDetectionContext,
+    ) -> Optional[FormatDetectionConfidence]:
+        # test maximum 10 annotation files only
+        ctr = 0
+        for file in context.require_files_iter("*.json"):
+            ctr += 1
+            with context.probe_text_file(
+                file, "Annotation format is not Segmentat-Anything format"
+            ) as f:
+                anno = json.load(f)
```

Comment:
Use `datumaro.util.parse_json()` for consistency.

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1005/commits/621167b72478f93d2c7f5814646d02040085ec33